src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -745,21 +734,15 @@ override public string CommandText
         ResCategoryAttribute(StringsHelper.ResourceNames.DataCategory_Data),
         ResDescriptionAttribute(StringsHelper.ResourceNames.TCE_SqlCommand_ColumnEncryptionSetting),
         ]
-        public SqlCommandColumnEncryptionSetting ColumnEncryptionSetting
-        {
-            get
-            {
-                return _columnEncryptionSetting;
-            }
-        }
+        public SqlCommandColumnEncryptionSetting ColumnEncryptionSetting => _columnEncryptionSetting;
 
         /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlCommand.xml' path='docs/members[@name="SqlCommand"]/CommandTimeout/*'/>
         [
         ResCategoryAttribute(StringsHelper.ResourceNames.DataCategory_Data),
         ResDescriptionAttribute(StringsHelper.ResourceNames.DbCommand_CommandTimeout),
         ]
         override public int CommandTimeout
```

Comment:
Could you swap `public' and 'override` here too?

Reply:
Done, and a few others too.